stubs/networkx/networkx/classes/digraph.pyi

```diff
@@ -34,9 +34,9 @@ class DiGraph(Graph[_Node]):
     @cached_property
     def in_edges(self) -> InEdgeView[_Node]: ...
     @cached_property
-    def in_degree(self) -> int | InDegreeView[_Node] | InMultiDegreeView[_Node]: ...
+    def in_degree(self) -> InDegreeView[_Node] | InMultiDegreeView[_Node]: ...
     @cached_property
-    def out_degree(self) -> int | OutDegreeView[_Node] | OutMultiDegreeView[_Node]: ...
+    def out_degree(self) -> OutDegreeView[_Node] | OutMultiDegreeView[_Node]: ...
```

Comment:
Do we need the "multi" views (e.g. for sub-classing)?

Reply:
See my comment here: https://github.com/python/typeshed/issues/14592#issuecomment-3207018505
tl;dr: Yes for LSP